CLM5-PDAF Test-Setup `nrw_5x`

jjokella opened the pull request.

jjokella committed:
TSMP-PDAF Setup `nrw_5x` (#193)

by @lstrebel

* setups: introduce nrw_5x setup

* supported_versions: introduce setup `nrw_5x`

jjokella committed:
nrw5_x: rename ksh file according to new conventional

- remove machine-specific files
- remove `$machine_setup` suffix from file name

jjokella committed:
nrw_5x: move `initSetup` to new conventions

jjokella committed:
nrw_5x: cleaned-up version of setup script

jjokella committed:
nrw_5x: (temporary) fix for unknown rundir in setup script

jjokella committed:
setups: input `deltaobs` only set for PDAF runs

jjokella committed:
setups: more PDAF inputs from setup script / PDAF SLURM runscripts

jjokella committed:
rundir fix: date is added later in `setup_tsmp`

lstrebel commented on bldsva/setups/nrw_5x/nrw_5x.ksh:
```diff
@@ -0,0 +1,90 @@
+#! /bin/ksh
+
+static_files=/p/scratch/cslts/shared_data/rcmod_TSMP-ref_SLTS/TestCases/nrw_5x
+
```
namelist_da=$rootdir/bldsva/setups/$refSetup/enkfpf.par

lstrebel commented on bldsva/setups/nrw_5x/nrw_5x.ksh:
```diff
@@ -0,0 +1,90 @@
+   rundir="$rootdir/run/${platform}_${combination}_${refSetup}"
+fi
+# End setting rundir
+
```
+mkdir -p $rundir

lstrebel reviewed (commented):
Need the namelist for this case to be set to enkfpf.par so the default from common_setup.ksh is not used. Also need to create the $rundir folder in this step so it already exists for the copy command a few lines down.

lstrebel committed:
Update nrw_5x.ksh

Change to DA namelist (enkfpf.par instead of DA-nl) and creation of $rundir directory.

jjokella committed:
nrw_5x: delete unused files

jjokella committed:
nrw_5x: make auxiliary setup-specific finalizeSetup routine

jjokella committed:
doc: remove mentions of `TSMP_pdaf` in the documentation

remove completely the outdated `branches.md`

jjokella committed:
setup_clm: remove code from build_interface and add in setup script

The clm5_0 specific commands are under condition

jjokella committed:
TSMP-PDAF: update introduction sentence

jjokella committed:
Merge branch 'master' into master-jjokella

jjokella committed:
Merge branch 'master-jjokella' into master-nrw_5x

jjokella committed:
Merge branch 'master' into master-nrw_5x
